packages/mui-docs/src/NProgressBar/NProgressBar.js

```diff
@@ -94,7 +38,52 @@ function NProgressBar(props) {
   return (
     <NoSsr>
       {props.children}
-      <GlobalStyles />
+      <GlobalStyles
+        styles={(theme) => ({
+          '#nprogress': {
+            direction: 'ltr',
+            pointerEvents: 'none',
+            position: 'fixed',
+            top: 0,
+            left: 0,
+            right: 0,
+            height: 2,
+            zIndex: (theme.vars || theme).zIndex.tooltip,
+            backgroundColor: (theme.vars || theme).palette.primary[200],
+            ...theme.applyDarkStyles({
+              backgroundColor: (theme.vars || theme).palette.primary[700],
+            }),
+            '& .nprogress-bar': {
+              position: 'fixed',
+              backgroundColor: (theme.vars || theme).palette.primary.main,
+              top: 0,
+              left: 0,
+              right: 0,
+              height: 2,
+            },
+            '& .nprogress-bar > div': {
+              position: 'absolute',
+              top: 0,
+              height: 2,
+              boxShadow: `${(theme.vars || theme).palette.primary.main} 1px 0 6px 1px`,
+              borderRadius: '100%',
+              animation: `${muiNProgressPulse} 2s ease-out 0s infinite`,
+            },
+            '& .nprogress-bar > div:first-child': {
```

Comment:
This warns on all the pages of the docs & website:

<img width="361" alt="Screenshot 2023-02-20 at 00 34 38" src="https://user-images.githubusercontent.com/3165635/219982198-8837a533-faa9-40d0-8906-5b628b01492d.png">

Could we fix it?

Reply:
Oops! here is the fix => https://github.com/mui/material-ui/pull/36263